proto/spacemesh/global_state_types.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+package spacemesh;
+import "spacemesh/types.proto";
+
+// Data items related to an account
+enum AccountDataFlag {
+    NONE = 0;
+    TRANSACTION_RECIEPT = 1; // tx receipt for a tx to or from an account
+    REWARD = 2; // reward awarded to an account
+}
+
+message AccountDataParams {
+    AccountId account_id = 1;
+    uint32 account_data_bit_field = 2; // A bit field of AccountDataFlag
+}
+
+message AccountDataQueryParams {
```

Comment:
No `min_layer`?

Reply:
Good question. See my comments about the queries in GlobalStateService. I didn't add min layer to some queries because this assumed that all of these data items will needed to be indexed by account id and and list support min layer ordering (or a layer index). I think we should have them w/o min layer for now until you find out how expensive it is to support the more robust version with min_layer for them...

Reply:
Okay, I'm happy to keep these simpler for now, until we know how hard implementation will be as you suggest.